### ISSUE TITLE
Training logs

### ISSUE THREAD

**Issue by letsmakeadeal** [uvavision/RerankingTransformer#5]
Hi, thanks for great work!
Can you please open access to your's training logs, if they are storing somewhere?

**Comment by fwtan** [comment]
Hi, I have graduated from UVA and do not have everything in hand now. I manage to find the log for [R101 + GLDv2](https://github.com/uvavision/RerankingTransformer/tree/main/RRT_GLD/logs/r101_gldv2_log) and [R50 + SOP (finetune)](https://github.com/uvavision/RerankingTransformer/tree/main/RRT_SOP/logs/finetune_log), which have been uploaded to this repo. Hope they could help.

The accuracy shown in the log for [R50 + SOP (finetune)](https://github.com/uvavision/RerankingTransformer/tree/main/RRT_SOP/logs/finetune_log) is higher than what we reported in the paper as the test images of this experiment run were directly resized to 224 x 224, instead of center-cropped to 224x224. We ran an extra evaluation for center-cropping and reported that number in the paper.
